### ISSUE TITLE
Notify all players (signups + waitlisted) when an event is dropped

### ISSUE THREAD

**Issue by dkapell** [neinteractiveliterature/intercode#7447]
Ace Tayloe says: Feature request (possibly a feature modification request): If a game gets canceled, send out an automatic email informing all players and waitlisters of the cancelation. (Source: A friend was waitlisted for High Tea, which got canceled; they didn't get a notification about the cancelation, and only found out today when they checked the site.)

**Comment by dkapell** [comment]
Dave says: There may be times when Con staff wishes to remove an event without notification, but maybe that should be an option on the drop event confirmation dialog.

**Comment by marleighnorton** [comment]
Yes, notifications make sense.  We should do that.  Usual rules (e-mail always, also text if enabled)
